wpilibj/src/shared/java/edu/wpi/first/wpilibj/command/Scheduler.java

```diff
@@ -195,30 +177,25 @@ public void run() {
     // Get button input (going backwards preserves button priority)
     if (m_buttons != null) {
       for (int i = m_buttons.size() - 1; i >= 0; i--) {
-        ((ButtonScheduler) m_buttons.elementAt(i)).execute();
+        m_buttons.get(i).execute();
```

Comment:
From line 177: `// Get button input (going backwards preserves button priority)`

Could also be written like:
```
      ListIterator<ButtonScheduler> iterator = m_buttons.listIterator();
      while (iterator.hasPrevious()) {
        iterator.previous().execute();
      }
```